### PR TITLE
Added changes to execute 4.x pipeline build from jenkins

### PR DESCRIPTION
**Pull request by manasagowri**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: pipeline/4/Jenkinsfile-tier-0.groovy [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                extensions: [[
+                    $class: 'SubmoduleOption',
+                    disableSubmodules: false,
+                    parentCredentials: false,
+                    recursiveSubmodules: true,
+                    reference: '',
+                    trackingSubmodules: false
+                ]],
+                submoduleCfg: [],
+                userRemoteConfigs: [[url: 'https://github.com/manasagowri/cephci.git']]
```
Comment (rakeshgm) [inline pipeline/4/Jenkinsfile-tier-0.groovy:53]: shouldn't this be `https://github.com/red-hat-storage/cephci`. you are using your fork here.

**Comment by rakeshgm** [comment]
looks like your s3 tests code needs to be updated. @psathyan has already sent the changes and its merged https://github.com/red-hat-storage/cephci/pull/424 .

**Comment by rakeshgm** [comment]
i am think that you only want to submit this commit. https://github.com/red-hat-storage/cephci/pull/429/commits/28adb9c226fc0d5cf551f6a7cd82be3c9f94ca41 right ?

File: pipeline/4/Jenkinsfile-tier-0.groovy [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                userRemoteConfigs: [[url: 'https://github.com/red-hat-storage/cephci.git']]
+            ])
+            script {
+                sharedLib = load("${env.WORKSPACE}/pipeline/vars/common.groovy")
+                sharedLib.prepareNode()
+            }
+        }
+    }
+
+    timeout(unit: "HOURS", time: 12) {
```
Comment (psathyan) [inline pipeline/4/Jenkinsfile-tier-0.groovy:62]: Could we reduce the time out? Let's start with a smaller limit

File: pipeline/4/Jenkinsfile-tier-0.groovy [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                 ]
+
+// Pipeline script entry point
+node(nodeName) {
+
+    timeout(unit: "MINUTES", time: 30) {
+        stage('Install prereq') {
+            checkout([
+                $class: 'GitSCM',
+                branches: [[name: '*/4.x_build']],
```
Comment (psathyan) [inline pipeline/4/Jenkinsfile-tier-0.groovy:42]: I believe this should be `*/master`

File: pipeline/5/Jenkinsfile-tier-0.groovy [hunk 4]
```diff
@@ -10,7 +10,6 @@ def testStages = ['deploy': {
                     stage('Deployment suite') {
                         script {
                             withEnv([
-                                "sutVMConf=conf/inventory/rhel-8.3-server-x86_64.yaml",
```
Comment (psathyan) [inline pipeline/5/Jenkinsfile-tier-0.groovy:13]: I prefer having this option available to the test suite... we can chose the default if it's not available

**Comment by manasagowri** [comment]
> i am think that you only want to submit this commit. [28adb9c](https://github.com/red-hat-storage/cephci/commit/28adb9c226fc0d5cf551f6a7cd82be3c9f94ca41) right ?

File: pipeline/vars/common.groovy [hunk 5]
```diff
@@ -37,7 +37,6 @@ def runTestSuite() {
     cmd += " --instances-name ${instanceName}"
 
     // Append test suite specific
-    cmd += " --inventory ${env.sutVMConf}"
```
Comment (psathyan) [inline pipeline/vars/common.groovy:40]: Please keep this statement. In case of not being set, we can use the default options.

**Pull request closed without merging**